business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/model/KieSeleniumTest.java

```diff
@@ -43,4 +48,16 @@ private static KieWbDistribution kieWbDistribution() {
                 .fromWarNameString(prop)
                 .orElseThrow(() -> new IllegalStateException("Invalid app.name='" + prop + "' Expecting business-central or business-monitoring"));
     }
+
+    public static int getHomepageLoadingTimeoutSeconds(final int defaultTimeout) {
+        final String timeout = System.getProperty("selenium.homepage.loading.timeout.seconds");
+        try {
+            return Integer.parseInt(timeout);
+        } catch (NumberFormatException nfe) {
+            LOG.warn("System property \'{}\' can not be parsed. Setting to default value: {}",
```

Comment:
I prefer to ignore value if empty (provide default one) and throw an exception if the value is not valid. Because if someone trying to set a value and provided incorrect value - it shouldn't be swallowed. And if value is empty there is no need to escalate exceptions.